packages/angular_devkit/build_angular/test/browser/styles_spec_large.ts

```diff
@@ -639,4 +683,49 @@ describe('Browser Builder styles', () => {
       expect(await files['styles.css']).toMatch(/\.one(.|\n|\r)*\.two(.|\n|\r)*\.three/);
     });
   });
+
+  extensionsWithImportSupport.forEach(ext => {
+    it(`adjusts relative resource URLs when using @import in ${ext} (global)`, async () => {
+      host.copyFile('src/spectrum.png', './src/more-styles/images/global-img-relative.png');
+      host.writeMultipleFiles({
+        [`src/styles-one.${ext}`]: tags.stripIndents`
+            @import "more-styles/styles-two.${ext}";
+          `,
+        [`src/more-styles/styles-two.${ext}`]: tags.stripIndents`
+            .two {
+              background-image: url(images/global-img-relative.png);
+            }
+          `,
+      });
+
+      const overrides = {
+        extractCss: true,
```

Comment:
For the sake of it, shouldn't we run the test without sourcemaps since this behaviour relies on them?

Reply:
Good idea.  Will do.